astropy/_dev/scm_version.py

```diff
@@ -1,10 +1,10 @@
 # Try to use setuptools_scm to get the current version; this is only used
 # in development installations from the git repository.
-import os.path as pth
+from pathlib import Path
 
 try:
     from setuptools_scm import get_version
 
-    version = get_version(root=pth.join("..", ".."), relative_to=__file__)
+    version = get_version(root=Path("../.."), relative_to=__file__)
```

Comment:
I think this is OS dependent. ``.parent`` or ``.parents``.

Reply:
done